<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="24" failed="1" total="25" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="CSAFP_TEST_AUTOMATION" duration-ms="8151" started-at="2020-10-14T06:15:15Z" finished-at="2020-10-14T06:15:24Z">
    <groups>
      <group name="requester">
        <method signature="US_CSAfp_fastpath.ripc_loginTest()[pri:1, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_loginTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_createRequestTest()[pri:2, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_createRequestTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_enterProjectAttributesTest()[pri:3, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_enterProjectAttributesTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_addMilestoneTest()[pri:4, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_addMilestoneTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_enterMilestoneDetailsTest()[pri:5, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_enterMilestoneDetailsTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.add_JRSSTest()[pri:6, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="add_JRSSTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.click_PeopleInvolvedTest()[pri:7, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="click_PeopleInvolvedTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.click_SuppliersTest()[pri:8, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="click_SuppliersTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_supplierQuestionsTest()[pri:9, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_supplierQuestionsTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_supplierEvaluationTest()[pri:10, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_supplierEvaluationTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_termsandcondition()[pri:11, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_termsandcondition" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.ripc_submitRequestTest()[pri:12, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_submitRequestTest" class="CSAFP_US.US_CSAfp_fastpath"/>
      </group> <!-- requester -->
      <group name="requester1">
        <method signature="US_CSAfp_fastpath.ripc_loginTest1()[pri:23, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_loginTest1" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.RIPC_reqSearchTest()[pri:24, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="RIPC_reqSearchTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.RIPC_openRequestTest()[pri:25, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="RIPC_openRequestTest" class="CSAFP_US.US_CSAfp_fastpath"/>
      </group> <!-- requester1 -->
      <group name="buyer">
        <method signature="US_CSAfp_fastpath.buyer_loginTest()[pri:13, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_loginTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_awaitingActionTest()[pri:14, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_awaitingActionTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_verifyProjectAttributesTest()[pri:15, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifyProjectAttributesTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_milestoneTest()[pri:16, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_milestoneTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_verifyPeopleInvolvedTest()[pri:17, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifyPeopleInvolvedTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_verifySuppliersTest()[pri:18, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifySuppliersTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_verifySupplierQuesTest()[pri:19, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifySupplierQuesTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_enterEvaluationCriteriaTest()[pri:20, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_enterEvaluationCriteriaTest" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_termsandcondition()[pri:21, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_termsandcondition" class="CSAFP_US.US_CSAfp_fastpath"/>
        <method signature="US_CSAfp_fastpath.buyer_SkipSupplier()[pri:22, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_SkipSupplier" class="CSAFP_US.US_CSAfp_fastpath"/>
      </group> <!-- buyer -->
    </groups>
    <test name="CSAFP-US_FASTPATH" duration-ms="8151" started-at="2020-10-14T06:15:15Z" finished-at="2020-10-14T06:15:24Z">
      <class name="CSAFP_US.US_CSAfp_fastpath">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="beforeClass" is-config="true" duration-ms="324" started-at="2020-10-14T11:45:16Z" finished-at="2020-10-14T11:45:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="ripc_loginTest()[pri:1, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_loginTest" duration-ms="7475" started-at="2020-10-14T11:45:16Z" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Wrappers.GenericWrappers_US.login(GenericWrappers_US.java:38)
	at CSAFP_US.US_CSAfp_fastpath.ripc_loginTest(US_CSAfp_fastpath.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_loginTest -->
        <test-method status="SKIP" signature="ripc_createRequestTest()[pri:2, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_createRequestTest" duration-ms="0" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_loginTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_createRequestTest()[pri:2, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_createRequestTest()[pri:2, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_createRequestTest -->
        <test-method status="SKIP" signature="ripc_enterProjectAttributesTest()[pri:3, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_enterProjectAttributesTest" duration-ms="0" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_createRequestTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_enterProjectAttributesTest()[pri:3, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_enterProjectAttributesTest()[pri:3, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_enterProjectAttributesTest -->
        <test-method status="SKIP" signature="ripc_addMilestoneTest()[pri:4, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_addMilestoneTest" duration-ms="0" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_enterProjectAttributesTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_addMilestoneTest()[pri:4, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_addMilestoneTest()[pri:4, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_addMilestoneTest -->
        <test-method status="SKIP" signature="ripc_enterMilestoneDetailsTest()[pri:5, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_enterMilestoneDetailsTest" duration-ms="0" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_addMilestoneTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_enterMilestoneDetailsTest()[pri:5, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_enterMilestoneDetailsTest()[pri:5, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_enterMilestoneDetailsTest -->
        <test-method status="SKIP" signature="add_JRSSTest()[pri:6, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="add_JRSSTest" duration-ms="20" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_enterMilestoneDetailsTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.add_JRSSTest()[pri:6, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.add_JRSSTest()[pri:6, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_JRSSTest -->
        <test-method status="SKIP" signature="click_PeopleInvolvedTest()[pri:7, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="click_PeopleInvolvedTest" duration-ms="0" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.add_JRSSTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.click_PeopleInvolvedTest()[pri:7, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.click_PeopleInvolvedTest()[pri:7, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_PeopleInvolvedTest -->
        <test-method status="SKIP" signature="click_SuppliersTest()[pri:8, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="click_SuppliersTest" duration-ms="0" started-at="2020-10-14T11:45:23Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.click_PeopleInvolvedTest" finished-at="2020-10-14T11:45:23Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.click_SuppliersTest()[pri:8, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.click_SuppliersTest()[pri:8, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_SuppliersTest -->
        <test-method status="SKIP" signature="ripc_supplierQuestionsTest()[pri:9, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_supplierQuestionsTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.click_SuppliersTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_supplierQuestionsTest()[pri:9, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_supplierQuestionsTest()[pri:9, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_supplierQuestionsTest -->
        <test-method status="SKIP" signature="ripc_supplierEvaluationTest()[pri:10, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_supplierEvaluationTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_supplierQuestionsTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_supplierEvaluationTest()[pri:10, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_supplierEvaluationTest()[pri:10, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_supplierEvaluationTest -->
        <test-method status="SKIP" signature="ripc_termsandcondition()[pri:11, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_termsandcondition" duration-ms="1" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_supplierEvaluationTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_termsandcondition()[pri:11, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_termsandcondition()[pri:11, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_termsandcondition -->
        <test-method status="SKIP" signature="ripc_submitRequestTest()[pri:12, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_submitRequestTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_termsandcondition" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_submitRequestTest()[pri:12, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_submitRequestTest()[pri:12, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_submitRequestTest -->
        <test-method status="SKIP" signature="buyer_loginTest()[pri:13, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_loginTest" duration-ms="0" depends-on-groups="requester" started-at="2020-10-14T11:45:24Z" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_loginTest()[pri:13, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods in group "requester"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_loginTest()[pri:13, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods in group "requester"
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_loginTest -->
        <test-method status="SKIP" signature="buyer_awaitingActionTest()[pri:14, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_awaitingActionTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_loginTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_awaitingActionTest()[pri:14, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_awaitingActionTest()[pri:14, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_awaitingActionTest -->
        <test-method status="SKIP" signature="buyer_verifyProjectAttributesTest()[pri:15, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifyProjectAttributesTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_awaitingActionTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_verifyProjectAttributesTest()[pri:15, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_verifyProjectAttributesTest()[pri:15, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_verifyProjectAttributesTest -->
        <test-method status="SKIP" signature="buyer_milestoneTest()[pri:16, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_milestoneTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_verifyProjectAttributesTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_milestoneTest()[pri:16, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_milestoneTest()[pri:16, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_milestoneTest -->
        <test-method status="SKIP" signature="buyer_verifyPeopleInvolvedTest()[pri:17, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifyPeopleInvolvedTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_milestoneTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_verifyPeopleInvolvedTest()[pri:17, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_verifyPeopleInvolvedTest()[pri:17, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_verifyPeopleInvolvedTest -->
        <test-method status="SKIP" signature="buyer_verifySuppliersTest()[pri:18, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifySuppliersTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_verifyPeopleInvolvedTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_verifySuppliersTest()[pri:18, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_verifySuppliersTest()[pri:18, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_verifySuppliersTest -->
        <test-method status="SKIP" signature="buyer_verifySupplierQuesTest()[pri:19, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_verifySupplierQuesTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_verifySuppliersTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_verifySupplierQuesTest()[pri:19, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_verifySupplierQuesTest()[pri:19, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_verifySupplierQuesTest -->
        <test-method status="SKIP" signature="buyer_enterEvaluationCriteriaTest()[pri:20, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_enterEvaluationCriteriaTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_verifySupplierQuesTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_enterEvaluationCriteriaTest()[pri:20, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_enterEvaluationCriteriaTest()[pri:20, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_enterEvaluationCriteriaTest -->
        <test-method status="SKIP" signature="buyer_termsandcondition()[pri:21, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_termsandcondition" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_enterEvaluationCriteriaTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_termsandcondition()[pri:21, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_termsandcondition()[pri:21, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_termsandcondition -->
        <test-method status="SKIP" signature="buyer_SkipSupplier()[pri:22, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="buyer_SkipSupplier" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_termsandcondition" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.buyer_SkipSupplier()[pri:22, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.buyer_SkipSupplier()[pri:22, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buyer_SkipSupplier -->
        <test-method status="SKIP" signature="ripc_loginTest1()[pri:23, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="ripc_loginTest1" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.buyer_SkipSupplier" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.ripc_loginTest1()[pri:23, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.ripc_loginTest1()[pri:23, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ripc_loginTest1 -->
        <test-method status="SKIP" signature="RIPC_reqSearchTest()[pri:24, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="RIPC_reqSearchTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.ripc_loginTest1" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.RIPC_reqSearchTest()[pri:24, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.RIPC_reqSearchTest()[pri:24, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RIPC_reqSearchTest -->
        <test-method status="SKIP" signature="RIPC_openRequestTest()[pri:25, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0]" name="RIPC_openRequestTest" duration-ms="0" started-at="2020-10-14T11:45:24Z" depends-on-methods="CSAFP_US.US_CSAfp_fastpath.RIPC_reqSearchTest" finished-at="2020-10-14T11:45:24Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method US_CSAfp_fastpath.RIPC_openRequestTest()[pri:25, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method US_CSAfp_fastpath.RIPC_openRequestTest()[pri:25, instance:CSAFP_US.US_CSAfp_fastpath@5e5792a0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RIPC_openRequestTest -->
      </class> <!-- CSAFP_US.US_CSAfp_fastpath -->
    </test> <!-- CSAFP-US_FASTPATH -->
  </suite> <!-- CSAFP_TEST_AUTOMATION -->
</testng-results>
